R/create_single_dose_dataset.R

```diff
@@ -0,0 +1,272 @@
+#' Pre-Defined Dose Frequencies
+#'
+#' These pre-defined dose frequencies are sourced from
+#' [CDISC](https://evs.nci.nih.gov/ftp1/CDISC/SDTM/SDTM%20Terminology.pdf). The
+#' number of rows to generate using `create_single_dose_dataset()` arguments
+#' `start_date` and `end_date` is derived from `DFFactor` and `DFUnit` with
+#' appropriate functions from `lubridate`.
+#'
+#' @details
+#' To see the entire table in the console, run `print(dose_freq_lookup)`.
+#'
+#' @seealso [create_single_dose_dataset()]
+#'
+#' @export
+#'
+#' @keywords dose_freq
+#'
+#' @rdname dose_freq_lookup
+#'
+dose_freq_lookup <- tibble::tribble(
+  ~NCICode, ~CDISCSubmissionValue,
+  "C64526", "1 TIME PER WEEK",
+  "C139179", "10 DAYS PER MONTH",
+#  "C176288", "2 TIMES PER CYCLE",
```

Comment:
What about the out commented lines? Shouldn't we remove them or define values for them?

Reply:
i was aiming for a lean working version of this that covers most cases. cycles can be of variable length, so i commented them out and did not define values for them. i think i would vote to just remove them if we don't like having commented lines